community/becoming-a-maintainer.md

```diff
@@ -1,3 +1,77 @@
 # Becoming a maintainer
 
-TODO
+## Introduction
+
+Our Maintainers are the people within Exercism that manage the different parts of our ecosystem. 
```

Comment:
```suggestion
Our maintainers are the people within Exercism that manage the different parts of our ecosystem. 
```

Reply:
I'm treating "Maintainer" as a proper noun, in this document (same as I do Concepts elsewhere for example)